src/org/rascalmpl/library/lang/rascal/tests/concrete/recovery/RascalRecoveryTests.rsc

```diff
@@ -56,40 +51,34 @@ test bool rascalIfMissingExpr() = checkRecovery(#FunctionDeclaration, "void f(){
 
 test bool rascalIfBodyEmpty() = checkRecovery(#start[Module], "module A void f(){1;} void g(){if(1){}} void h(){1;}", ["{", "} "]);
 
-// Not working yet:
-/*
-test bool rascalMissingOpeningParen() {
-    Tree t = parseRascal("module A void f){} void g() { }");
-
-    println("error text: <getErrorText(findFirstError(t))>");
-    return getErrorText(findFirstError(t)) == "a}";
-}
-
-test bool rascalFunFunMissingCloseParen() {
-    Tree t = parseRascal("module A void f(){void g({}} void h(){}");
-
-    println("error text: <getErrorText(findFirstError(t))>");
-    return getErrorText(findFirstError(t)) == "a}";
-}
-
-test bool rascalIfMissingOpeningParen() {
-    Tree t = parseRascal("module A void f(){if 1){}}", visualize=false);
-
-    println("error text: <getErrorText(findFirstError(t))>");
-    return getErrorText(findFirstError(t)) == ";";
-}
-
-test bool rascalIfMissingCloseParen() {
-    Tree t = parseRascal("module A void f(){if(1{}}", visualize=false);
-
-    println("error text: <getErrorText(findFirstError(t))>");
-    return getErrorText(findFirstError(t)) == ";";
-}
-
-test bool rascalIfMissingSemi() {
-    Tree t = parseRascal("module A void f(){if (true) {a}}");
-
-    println("error text: <getErrorText(findFirstError(t))>");
-    return getErrorText(findFirstError(t)) == ";";
-}
-*/
\ No newline at end of file
+test bool rascalMissingOpeningParen() = checkRecovery(#start[Module], 
+"module A
+void f) {
+    }
+    void g() { 
+    }", ["}
+",") {"]);
+
+test bool rascalFunFunMissingCloseParen() = checkRecovery(#start[Module], "module A void f(){void g({}} void h(){}", ["void g({}", "} "]);
+
+test bool rascalIfMissingOpeningParen() = checkRecovery(#start[Module], 
+"module A void f() {
+  if 1) {
+    1;
+  }}", ["1) "]);
+
+test bool rascalIfMissingCloseParen() = checkRecovery(#start[Module], 
+"module A
+void f() {
+  if (1 {
+    1;
+  }}", ["1 "]);
+
+test bool rascalIfMissingSemi() = checkRecovery(#start[Module], 
+"module A
+void f() {
```

Comment:
Very nit (possibly also in other places):

```suggestion
test bool rascalIfMissingSemi() = checkRecovery(#FunctionDeclaration, 
"void f() {
```

Would save just a tiny bit of noise in the tests